menas/src/main/resources/application.properties.template

```diff
@@ -68,9 +68,9 @@ menas.environment=Dev
 
 menas.spark.master=local[1]
 
-# The template for Lineage (Spline) display. If empty Lineage button is not shown.
+# The api gateway for Lineage (Spline) display. If empty Lineage button is not shown.
 # Avoiding the http(s) protocol will cause that the same protocol will be used as for Menas itself, avoiding issues
-spline.urlTemplate=//localhost:8080/spline/dataset/lineage/_search?path=%s&application_id=%s
+menas.lineage.readApiUrl=//localhost:8080/spline-gw/consumer
```

Comment:
what about the spline ref on line 125?

Reply:
it has been my belief, I had removed that in the Menas to Oozie integration ticket (#1126 ). Ill double-check.

Reply:
Yeah, it's not present anymore in `https://github.com/AbsaOSS/enceladus/blob/feature/735-integrate-spline-0-4-x/menas/src/main/resources/application.properties.template`
Not sure why it's not reflected in this PR. Maybe because of the unresolved conflicts... 🤔